src/main/java/com/openshift/jenkins/plugins/freestyle/BaseStep.java

```diff
@@ -236,11 +237,11 @@ public boolean perform(String filename) throws IOException,
     }
 
     protected boolean standardRunOcCommand(final AbstractBuild build,
-            final TaskListener listener, String verb, List verbArgs,
-            List userArgs, List options, List verboseOptions)
+            final TaskListener listener, boolean skipTLSVerify, String caPath, String verb, List verbArgs,
```

Comment:
Missed this last time, but noticed it this time, and then I checked out the branch and searched in eclipse to be sure .... for every call to this method, skipTLSVerify is false and caPath is ""

I think you need to parse the "Advanced Arguments" for the associated `oc` arguments and then set these parameters accordingly.